web-app/src/app/tabs/race/race-graph/race-graph.component.css

```diff
@@ -0,0 +1,43 @@
+ 
+ #raceGraphContainer{
```

Comment:
Is there a specific reason we are doing the CCS by id on the graph? If not, we should use the regular class groupings instead.

Reply:
]The idea was for the styling to only affect the section where the actual graph was and not all other ones (which may appear later on)

Reply:
I don't think we'll have any other cases where we'll have a graph on this page, but if we do I think these styling can stay consistent. Let's use the regular `.` prefix instead.

Reply:
Alright